normandy/base/utils.py

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
I usually do too, but Bedrock was specifically asked to add them, so I assumed it'd be a thing to do. But I'll remove them.